pkg/pipelines/tekton_checks.go

```diff
@@ -0,0 +1,113 @@
+package pipelines
+
+import (
+	"k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	"k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/typed/apiextensions/v1beta1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+// These CRDs names are checked to confirm that Tekton Pipelines/Triggers has been installed.
+var requiredCRDNames = []string{
+	"pipelines.tekton.dev", "pipelineresources.tekton.dev", "pipelineruns.tekton.dev",
+	"triggerbindings.tekton.dev", "triggertemplates.tekton.dev",
+	"clustertasks.tekton.dev", "taskruns.tekton.dev", "tasks.tekton.dev",
+}
+
+// checkInterface contains a method to return whether Tekton is installed
+type checkInterface interface {
```

Comment:
Generally in Go, [single method interfaces are named with `-er`](https://golang.org/doc/effective_go.html#interface-names).

how about `installChecker` or maybe `tektonInstallChecker` ?

Reply:
Actually, maybe even just `installedCRDChecker` given that there's nothing particularly Tekton specific about it?